pkg/console/subresource/configmap/configmap_test.go

```diff
@@ -112,19 +139,88 @@ func TestDefaultConfigMap(t *testing.T) {
 				BinaryData: nil,
 			},
 		},
+		{
+			name: "Test new Config Map that would come from managed namespace",
```

Comment:
- `Test generating default configmap without managed config override`
- `Test generating configmap with managed config to override branding`

Lets not use `New`.  Its new today, but won't be new forever.